<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCGUI</class>
 <widget class="QMainWindow" name="FCGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCGUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../MEDIA/制图/1x/画板 1-m.jpg</normaloff>../../../../../MEDIA/制图/1x/画板 1-m.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Calculator</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>空气阻力系数 单位N*s^2/m^2 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="airres"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>物体质量 单位kg </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mass"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>初速(m/s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="startv"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>总计算时间(ms)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tott"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>计算时间精度(ms)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timejd"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>输出时间精度(ms)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>输出拟合数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Ready. Powered by:UIC(https://kechuang.org/u/77939)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start">
          <property name="text">
           <string>开始计算</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;F-Calc GUI Edition &lt;a href=&quot;https://www.kechuang.org/t/84497&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://www.kechuang.org/t/84497&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;计算结果可能会有误差，且和精度有关。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;由UIC编写(&lt;/span&gt;&lt;a href=&quot;https://www.kechuang.org/u/77939&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#0000ff;&quot;&gt;https://kechuang.org/u/77939)&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:10px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:1.1%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-size:22.5pt; font-weight:600; font-style:italic; color:#282c37;&quot;&gt;UIC TECH&lt;/span&gt; &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;hr width=&quot;100%&quot;/&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-style:italic; color:#282c37;&quot;&gt;Hangzhou High School&lt;/span&gt; &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-style:italic; color:#282c37;&quot;&gt;Senior 1 Student&lt;/span&gt; &lt;/p&gt;
&lt;hr width=&quot;100%&quot;/&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-size:13.5pt; font-style:italic; color:#282c37;&quot;&gt;Physics&lt;/span&gt; &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-size:13.5pt; font-style:italic; color:#282c37;&quot;&gt;Infomatics&lt;/span&gt; &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-size:13.5pt; font-style:italic; color:#282c37;&quot;&gt;Aerospace&lt;/span&gt; &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-size:13.5pt; font-style:italic; color:#282c37;&quot;&gt;Inertial Navigation&lt;/span&gt; &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Arial,sans-serif'; font-size:13.5pt; font-style:italic; color:#282c37;&quot;&gt;Energetic Materials&lt;/span&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit">
      <property name="text">
       <string>退出程序</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>FCGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
